js/src/duplicate-post-edit-script.js

```diff
@@ -80,6 +84,21 @@ class DuplicatePost {
 		return false;
 	}
 
+	/**
+	 * Determines whether a Rewrite & Republish copy can be republished.
+	 *
+	 * @return bool Whether the Rewrite & Republish copy can be republished.
+	 */
+	is_copy_allowed_to_be_republished() {
```

Comment:
Possibly rename to `isCopyAllowedToBeRepublished` to be consistent with the other function names in the js code?

Reply:
Hm I think our standards require "snake case format" for PHP method names.

Reply:
But this is not a PHP method name right?

Reply:
Ahhh this is the JS file! Sorry 🤦 Then yes, sure!

Reply:
👍